sites/public/src/pages/listing/[id]/[slug].tsx

```diff
@@ -70,7 +71,11 @@ export default function ListingPage(props: ListingProps) {
         <title>{pageTitle}</title>
       </Head>
       <MetaTags title={listing.name} image={metaImage} description={metaDescription} />
-      <ListingView listing={listing} jurisdiction={props.jurisdiction} />
+      {process.env.showNewSeedsDesigns ? (
+        <ListingViewSeeds listing={listing} jurisdiction={props.jurisdiction} />
```

Comment:
Rather than the new design being named `ListingViewSeeds`, could it just being `ListingView` and the old one renamed to `ListingViewDeprecated` or something like that? (Apologies if this was already discussed and I missed it.)

Reply:
Good q! Tbh I didn't do it here bc even with `git mv` the history was getting super wonky bc the new file had the same name as the old one and it just made it harder to review. I like the renaming, I'll follow w this one! https://github.com/bloom-housing/bloom/issues/4646